aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -510,6 +708,18 @@ module aptos_framework::account {
 
         // Update the `OriginatingAddress` table.
         update_auth_key_and_originating_address_table(addr, account_resource, new_auth_key);
+
+        if (to_scheme == MULTI_ED25519_SCHEME) {
```

Comment:
What is this change in rotate_authentication_key for if there's a separate flow for that for keyless accounts?

Reply:
Because we want to do indexing so we want to emit KeyRotationToMultiPublicKey anytime we have a rotation to a multikey (assuming we have the information to do so)

Reply:
I'm confused - does this change belong here if this PR is strictly about keyless accounts? If not, I'd suggest a separate PR

Reply:
Will move to separate PR